src/savegame_json.cpp

```diff
@@ -2948,23 +2949,37 @@ void mission::serialize( JsonOut &json ) const
     json.end_object();
 }
 
+void activity_reason_info::serialize( JsonOut &json ) const
+{
+    json.start_object();
+    json.member( "reason", reason );
```

Comment:
```suggestion
    json.member( "reason" );
```
This writes the key, and the next line writes the value.